src/nb_clean/__init__.py

```diff
@@ -227,6 +234,10 @@ def clean_notebook(
             cell["execution_count"] = None
             if not preserve_cell_outputs:
                 cell["outputs"] = []
+            else:
+                for output in cell["outputs"]:
+                    if output.get("execution_count"):
```

Comment:
Let's change this to `if "execution_count" in output:`, for the same reason as above.

Reply:
Changed to `if output.get("execution_count") is not None`